src/cartogram_info/read_csv.cpp

```diff
@@ -329,21 +419,25 @@ void CartogramInfo::read_csv()
         << "ERROR: CSV with >= 2 columns (IDs, target areas) required. Some "
            "rows in your CSV may not have values for all columns"
         << std::endl;
-      _Exit(17);
+      std::exit(17);
     }
 
     const std::string id = row[static_cast<size_t>(id_col)].get();
     const std::string area_as_str = row[static_cast<size_t>(area_col)].get();
     check_validity_of_area_str(area_as_str);
 
-    const std::string color =
-      (color_col != csv::CSV_NOT_FOUND) ? row[static_cast<size_t>(color_col)].get() : "";
+    const std::string color = (color_col != csv::CSV_NOT_FOUND)
+                                ? row[static_cast<size_t>(color_col)].get()
+                                : "";
 
-    const std::string label =
-      (label_col != csv::CSV_NOT_FOUND) ? row[static_cast<size_t>(label_col)].get() : "";
+    const std::string label = (label_col != csv::CSV_NOT_FOUND)
+                                ? row[static_cast<size_t>(label_col)].get()
+                                : "";
 
     const std::string inset_pos_as_str =
-      (inset_col != csv::CSV_NOT_FOUND) ? row[static_cast<size_t>(inset_col)].get() : "C";
+      (inset_col != csv::CSV_NOT_FOUND)
+        ? row[static_cast<size_t>(inset_col)].get()
+        : "C";
```

Comment:
Good catch @SaifKhan21 , we should probably set our formatters to auto-format based on the `.clang-format` file included too @nihalzp, I think a lot of files are out of format.